.github/actions/common.sh

```diff
@@ -68,7 +68,9 @@ warn() {
 start_group Deduce Common Environment Variables
 export LDMX_GOLD_LABEL_FILE=${GITHUB_WORKSPACE}/.github/actions/validate/gold_label
 export LDMX_BASE=$(cd ${GITHUB_WORKSPACE}/../ && pwd)
+export LDMX_VERSION_FILE=${LDMX_BASE}/VERSION
```

Comment:
```suggestion
export LDMX_VERSION_FILE=${LDMX_BASE}/ldmx-sw/VERSION
```

`LDMX_BASE` is set to ldmx-sw's parent directory so we need to re-enter ldmx-sw to find VERSION.